### DISCUSSION TITLE
Slingbox 500 - New Setup

### DISCUSSION THREAD

**Discussion by Sure4Thing** [GerryDazoo/Slinger#290] [Q&A]
Hey All:

I love tinkering and figured Id see if I can get this 500 to work with Slinger.  I did a factory reset and now I have the slow blinking sling icon on the box.  

I started up my slingplayer 1.5.1 and it is showing a blue starting screen but it never changes.  I also tried using the Slinger windows version.  It finds my slingbox no problem and doesnt seem to have any issue using admin as the password

My question is 2 fold:

1.  Do I need to have a source plugged into the input for it to actually start
2. When I run the slinger server and try and access the URL I get a continual error saying slingbox in use! Kicking off other user.  I figure it is because it is trying to load settings and isnt setup?

Ive never used a slingbox before but figured yall would probably know best.
![slinger](https://user-images.githubusercontent.com/65963251/201558044-62942d06-ed34-4fa4-93c7-cbc6b5a818cd.JPG)
![slingplayer](https://user-images.githubusercontent.com/65963251/201558045-6e7a8376-430c-43f8-9191-c26a9fa6c189.JPG)

**Comment by sp222222** [comment]
you didn't read the readme at the beginning. Since you did a hard reset on the 500, you now have no way of knowing what the admin password is to it. Also, the 500 does not use that setup wizard you have pictured. Your 500 adminstrator password is some jumble of 15 characters. Good luck finding another piece of hardware that's older and can be used with that wizard app you have open (the tuner, AV, solo, PRO or PROHD versions). This might not be your hobby.

**Reply by sp222222** [reply]
https://github.com/GerryDazoo/Slinger/files/9997979/Step-by-step.pdf

**Reply by Sure4Thing** [reply]
I read the readme and I also saw on reddit at least one user performed the factory reset and it was successful using [admin on the 500](https://www.reddit.com/r/slingbox/comments/whwewc/comment/ipqe8tf/?utm_source=share&utm_medium=web2x&context=3)

I also have a Pro-HD here but the power supply went bad so I have a new one inbound.

I was also considering using the Mac Version that [appears to be working ](https://www.reddit.com/r/slingbox/comments/yfpw3s/comment/iup1p5j/?utm_source=share&utm_medium=web2x&context=3)

While I appreciate your efforts to dissuade me from learning something new, the Slinger app tells me the password is incorrect if I use a random password but I dont get the same error if I use the admin password so I was curious as to the setup of these devices.  I am trying to find out if I need to have an input source configured for the slingbox to try and set itself up or not.

![slinger_adminpw](https://user-images.githubusercontent.com/65963251/201561127-c1fcf454-568f-461a-b7a3-4f60b6c50ac2.JPG)
![slinger_incorrectPW](https://user-images.githubusercontent.com/65963251/201561128-79d7f4a2-dc48-422f-b7c1-b77fea45d57d.JPG)

**Reply by RandomMan719** [reply]
Why are you changing the slingbox port# in the first screenshot you have 5101 and the second one you have 5204?  I don't have a 500 model, but I do have a 350 and I believe the default port for the 350 and 500 is port 5201.  Try that.
![Default ports](https://user-images.githubusercontent.com/116921195/201570960-e2126003-f0b8-47ce-bcec-0b56fe31bf21.PNG)

admin for the password, does seem to be working, so that's a plus.

**Reply by rbf121** [reply]
I was the person you mentioned that was able to get it to work but this was before the shutdown and I am not brave enough to try it again now.  I had another comment on reddit with more findings: [here](https://www.reddit.com/r/slingbox/comments/y6oq1d/comment/it0rj8u/) 

These instructions are for a Slingbox 500 using the on screen instructions when you plug the Slingbox into your TV and use the provided remote control.

Pasting comments here as well:
I just tested this with a 500 and I'm able to get video after a factory reset.

- To test this, I blocked slingbox.com and sling.com from my router.
- It appears that the automatic update comes from some path on sling.com because blocking slingbox.com alone still allowed a firmware update.
- After factory reset, it will ask you to connect to ethernet or wifi. I selected via ethernet
- At this point, it requires you to update your slingbox, it will try to get an auto update but fail. The 500 onscreen display will give you only 2 options: try again or connect to another network. If you try to connect to the slingbox now, it will keep kicking you off, saying that the slingbox is in use.
- At this point, unplug slingbox and plug it back in
- Upon reboot, it will reach a screen that says "Slingbox updated" which obviously is not possible since I blocked sling.com and there was no way an update could be applied in the amount of time the reboot took
- Now restart your slinger app, the log should find a "My slingbox" listing the port and local ip address. set config.ini to include the local ip address, port and "admin" as your password.
- Restart slinger again
- Connect to the slinger stream and you will have video. I tested using component input (VideoSource 1)

Curious if anyone else can reproduce these steps. That being said, I will probably still update this slingbox to load the correct remote bin file.

But the key thing was to set the 500 up via the remote control to the point where it wants to update the firmware.  Then physically unplug the 500 once you got past some of the onscreen instructions.  When it reboots again, you should be able to use "admin" to connect to it.  the port would have changed as well.

**Reply by rbf121** [reply]
Also worth noting that in the current state, you will not be able to configure IR Remote codes so you can only open the stream unless you can figure out a way to load new .bin files.  

Someone posted some javascript code on reddit which looked promising but someone would need time to figure it out.

**Reply by RandomMan719** [reply]
Bullet point "After factory reset, it will ask you to connect to ethernet or wifi. I selected via ethernet"

Are you talking about using Slingplayer Desktop the new version?  Doesn't mention the software you are using.
![slingplayer desktop](https://user-images.githubusercontent.com/116921195/201574242-8fde9380-16ae-4e15-bf64-bccf7221ae64.PNG)

**Reply by rbf121** [reply]
Sorry, actually neither, I should have mentioned this was done using the on screen instructions when you plug the 500 into your TV via HDMI.

**Reply by RandomMan719** [reply]
So it is a builtin setup process, I have never set one up like that, I have a 350. I might check out that process.

Okay that might help, Sure4thing, get it working.

**Reply by Sure4Thing** [reply]
> Why are you changing the slingbox port# in the first screenshot you have 5101 and the second one you have 5204? I don't have a 500 model, but I do have a 350 and I believe the default port for the 350 and 500 is port 5201. Try that. ![Default ports](https://user-images.githubusercontent.com/116921195/201570960-e2126003-f0b8-47ce-bcec-0b56fe31bf21.PNG)
> 
> admin for the password, does seem to be working, so that's a plus.

Whenever I dont specify the IP and port slinger does a scan of my network and identifies port 5101.  When I try it as 5201 it doesnt seem to pick up on it.
![slinger_boxfound](https://user-images.githubusercontent.com/65963251/201714116-d3d3af54-d570-43b4-b870-8bc926809fa2.JPG)

**Reply by Sure4Thing** [reply]
> I was the person you mentioned that was able to get it to work but this was before the shutdown and I am not brave enough to try it again now. I had another comment on reddit with more findings: [here](https://www.reddit.com/r/slingbox/comments/y6oq1d/comment/it0rj8u/)
> 
> These instructions are for a Slingbox 500 using the on screen instructions when you plug the Slingbox into your TV and use the provided remote control.
> 
> Pasting comments here as well: I just tested this with a 500 and I'm able to get video after a factory reset.
> 
> * To test this, I blocked slingbox.com and sling.com from my router.
> * It appears that the automatic update comes from some path on sling.com because blocking slingbox.com alone still allowed a firmware update.
> * After factory reset, it will ask you to connect to ethernet or wifi. I selected via ethernet
> * At this point, it requires you to update your slingbox, it will try to get an auto update but fail. The 500 onscreen display will give you only 2 options: try again or connect to another network. If you try to connect to the slingbox now, it will keep kicking you off, saying that the slingbox is in use.
> * At this point, unplug slingbox and plug it back in
> * Upon reboot, it will reach a screen that says "Slingbox updated" which obviously is not possible since I blocked sling.com and there was no way an update could be applied in the amount of time the reboot took
> * Now restart your slinger app, the log should find a "My slingbox" listing the port and local ip address. set config.ini to include the local ip address, port and "admin" as your password.
> * Restart slinger again
> * Connect to the slinger stream and you will have video. I tested using component input (VideoSource 1)
> 
> Curious if anyone else can reproduce these steps. That being said, I will probably still update this slingbox to load the correct remote bin file.
> 
> But the key thing was to set the 500 up via the remote control to the point where it wants to update the firmware. Then physically unplug the 500 once you got past some of the onscreen instructions. When it reboots again, you should be able to use "admin" to connect to it. the port would have changed as well.

Im going to run through this now!  The only issue I may have is I dont have a remote control for this box.

**Reply by GerryDazoo** [reply]
Slinger doesn't not assign or modifiy port numbers. That's done by the Sling code. 5201 is jaust a default and I've seen many different ones. If my code scans your network and says it's port 5101 use 5101. That info is coming directly out of the slingbox.

**Comment by RandomMan719** [comment]
Sure4Thing
I am pretty sure you don't have to have a source plugged into the input for it to start, but the source has to be configured (as well as the remote control) prior to using Slinger, with Slingplayer software (Slingplayer Desktop, for the 500 model, not 1.5 or 2.0)  So since that doesn't work anymore because you have to login into your account (servers shutdown), I don't think you can configure a source, at the moment, on that model.  Maybe you will be the one to figure out how to do it, and let the community know, best of luck.

PS, I have fixed many Slingbox power supplies, they have failed a lot on my ProHD's.  But, usually it is just a bad capacitor or two, that you desolder and replace with new caps.  The caps usually look swelled up, when they are bad.

**Comment by sp222222** [comment]
gerry will admit that post you linked to is far from correct…..stick with
proHD and you won’t go wrong. see you in a few days once power supply shows
up

in the meantime, please read

https://github.com/GerryDazoo/Slinger/files/9997979/Step-by-step.pdf



regards

**Reply by RandomMan719** [reply]
I think the guide needs to be updated, because the password server is shutdown, I believe, I can't connect to it.  So the first step doesn't work.  Maybe it should start with, Turn Off your slingbox and block it from the Internet so it doesn't get bricked.

**Comment by GerryDazoo** [comment]
To weight in here.
1) You never should have done a factory reset unless you had exhausted all the methods reported to find the password. But that is under the bridge now.
2) After the reset the default password in "admin"
3) You don't need a video source connected but then I would ask, what's the point?
4) The integrated IR blaster and our web-based remote control won't work because we don't have the software yet to allow us to configure it after a factory reset.
5) That PDF wasn't written by me and was "pre-shutdown"and is not maintained by me. I'm sorry now I re-posted it from the Facebook group, but I'm getting tired of all the discussions that start "I didn't read the instructions and made up stuff as I went along and now it doesn't work". Especially the ones that say "I put this wrong URL into VLC or their favourite media player and it doesn't work. 
6) I wish I could find my slingbox remote.

**Reply by RandomMan719** [reply]
Gerry, thank you for some clarification.  I haven't seen any methods to find the admin password, besides the server, that is now shutdown.  Where can I find those?  Also I haven't done any factory resets post Nov. 9th, and will leave that as a last resort.  I am not too worried about the guide, I just think it needs to be updated for "post-shutdown."  So people don't brick more boxes.  
Slinger is pretty easy to setup, if you just read the available info, and have basic networking knowledge.
Also I have a remote that went to a sling catcher box (the box is dead, remote still works), what do you need the remote for?  And would this one help you?
![Remote](https://user-images.githubusercontent.com/116921195/201719035-e78794d6-d6bc-49d4-a4a5-6c2fdd40c564.jpg)

**Reply by JoeM2016** [reply]
I can find mine in the Slingplayer Desk top log. Mine is located at:
C:\Users\joe_m\AppData\Roaming\Sling Media\Slingplayer Desktop\Logs\console

open text file and search from the end of the file for
USER":"admin","PASSWORD
May also want to get:
FINDER_ID

**Comment by sp222222** [comment]
I am not believing admin works on a 500 for the password for administrator.
and no way I chance the working ones I have now

**Reply by DoogieHowitzerWMD** [reply]
I know the admin password on the 500 is changed after performing a factory reset, but I've only verified different passwords on a box after it had already performed the initial firmware updates from the remote setup. I couldn't say what the status of the password was prior to downloading the updates.

**Reply by RandomMan719** [reply]
I was looking through my Slingplayer Desktop log and for my 350, after a factory reset, the password is set to admin.  But nothing is configured, so you couldn't use it with Slinger, when you configure it and it does the updates, it gets the 15 character password.  Also in the log when configured it shows "IS_CONFIGURED":-1

**Not configured**
11:2:7 738 : 5.0.0.83 : INFO :: DeLoreanLib :: Utils :: setConnectionParamsToConfig...Box object : {"IP":"192.168.2.115","PORT":5201,"USER":"admin","PASSWORD":"admin","FINDER_ID":"3EC361FB6D6C3D019A261A6F35BBF30F","NAME":"My Slingbox","PRODUCT_ID":17,"IS_CONFIGURED":0,"IS_OWNER":true,"IS_SAVED":0}

**Configured**
10:44:31 807 : 5.0.0.83 : INFO :: DeLoreanLib :: Utils :: setConnectionParamsToConfig...Box object : {"USER":"admin","PASSWORD":"juzE5eRcBqQLoc2","FINDER_ID":"3EC361FB6D6C3D019A261A6F35BBF30F","NAME":"Conference Room 350","PRODUCT_ID":17,"IS_CONFIGURED":-1,"IS_OWNER":true,"IS_SAVED":1}

**Reply by Sure4Thing** [reply]
I noticed the exact same behavior where the admin/admin is available for that short while but it wont do anything with that.  I am trying to see if there is a way extract out that password anywhere.  Id love to get some examples of 15 character passwords and see if I can generate some kind of machine learning algorithm to reverse engineer the algorithm it uses to generate keys

**Reply by JoeM2016** [reply]
Line from my working SB500 console log is:
"PRODUCT_ID":18,"IS_CONFIGURED":1 
Note no (-) in front of 1

**Reply by rbf121** [reply]
> I noticed the exact same behavior where the admin/admin is available for that short while but it wont do anything with that. I am trying to see if there is a way extract out that password anywhere. Id love to get some examples of 15 character passwords and see if I can generate some kind of machine learning algorithm to reverse engineer the algorithm it uses to generate keys

I'm able to decrypt the hashed passwords, the way I did it is very hacky but I just tested it and it worked.  If someone has a hashed password and want me to test it, let me know.

**Reply by GerryDazoo** [reply]
rbf: What hashed passwords are you referring to? The one in the log files E1:xxxxxx?  Or just the actual 15 character ones?
;password=lxk0VFQ8CY8j8sM
;password=m8h1a3l383fi74y 
password=3s4z2141a121ujn

**Reply by rbf121** [reply]
I was referring to the 64 character one after "E1:"

**Reply by RandomMan719** [reply]
I am out of town, but I will send you one to test when I get back.

**Comment by sp222222** [comment]
did you trt stepping through the setup using the sling remote they give to
those
boxes? mine from ebay this summer did not but the one I had with my 500
worked on the second 500 and I added the hdmi input that way (using
splitter off cablebox).

**Reply by Sure4Thing** [reply]
No unfortunately I dont have the remote for my 500

It appears that immediately after a factory reset the admin/admin is partially available but if you reboot it after that it gets changed to something else probably that 15 character random string.

I am working on decompiling the firmware for a 500 box and tomorrow or the next day will dump the EEPROM while it is running and see if it stores the password in there

**Reply by RandomMan719** [reply]
No, I haven't, I bought the 350 off ebay probably 4 years ago or more, so those are old logs.  I don't know what remote you are talking about, I don't think it came with a remote, the only remote I have (image is above) is for the slingcatcher, I think.

**Comment by sp222222** [comment]
you can have my keys to my bricked boxes. although they are all the same
and none are newer set

**Reply by Sure4Thing** [reply]
I mean if they were previously generated it would make sense to include them in the sample set.  Its a long shot for sure but definitely possible

**Comment by rbf121** [comment]
Someone just posted a video over on reddit showing the steps on a 500 using a remote control. [Reddit link](https://www.reddit.com/r/slingbox/comments/yvkowq/i_figured_out_a_way_to_use_a_reset_slingbox_on/)  

Seems to be similar steps I took before shutdown (listed somewhere above in the thread), requires the 500 remote control.

**Comment by sp222222** [comment]
if you cracked the code on this it’s amazing. 📺 📺 📺

and is it possible Gerry to add that to your slinGER script as another line
in the config that user supplies the E1 sixty-four string and the log file
which shows slingboxes on local network spits the admin file out or better
yet , spits it into the actual config line that has password=

**Reply by sp222222** [reply]
oh , the 64 string works in the password slinger field already

**Comment by sp222222** [comment]
![image](https://user-images.githubusercontent.com/110786843/219786651-386bac7e-b12c-4942-84a9-13da4b970489.png)
